seed/tests/test_meter_views.py

```diff
@@ -147,3 +145,102 @@ def test_update_meter(self):
+
+        # write a few values to the database
+        for values in [("2022-01-05T05:00:00Z", "2022-01-05T06:00:00Z", 6.0),
+                       ("2022-01-05T06:00:00Z", "2022-01-05T07:00:00Z", 12.0),
+                       ("2022-01-05T07:00:00Z", "2022-01-05T08:00:00Z", 18.0), ]:
+            payload = {
+                "start_time": values[0],
+                "end_time": values[1],
+                "reading": values[2],
+                "source_unit": "Wh (Watt-hours)",
+                # conversion factor is required and is the conversion from the source unit to kBTU (1 Wh = 0.00341 kBtu)
+                "conversion_factor": 0.00341,
+            }
+
+            response = self.client.post(url, data=json.dumps(payload), content_type='application/json')
+            self.assertEqual(response.status_code, 201)
+            self.assertEqual(response.json()['reading'], values[2])
+
+        # read all the values from the meter and check the results
+        response = self.client.get(url, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.json()), 3)
+
+    def test_delete_meter_readings(self):
+        # would be nice nice to make a factory out of the meter / meter reading requests
+        property_view = self.property_view_factory.get_property_view()
+        url = reverse('api:v3:property-meters-list', kwargs={'property_pk': property_view.id})
+
+        payload = {
+            'type': 'Natural Gas',
+            'source': 'Manual Entry',
+            'source_id': '9876543210',
+        }
+
+        response = self.client.post(url, data=json.dumps(payload), content_type='application/json')
+        meter_pk = response.json()['id']
+
+        # create meter reading  property-meter-readings-list
+        url = reverse('api:v3:property-meter-readings-list', kwargs={'property_pk': property_view.id, 'meter_pk': meter_pk})
+
+        payload = {
+            "start_time": "2022-01-05T05:00:00Z",
+            "end_time": "2022-01-05T06:00:00Z",
+            "reading": 10,
+            "source_unit": "kBtu (Thousand BTU)",
+            # conversion factor is required and is the conversion from the source unit to kBTU (1 Wh = 0.00341 kBtu)
+            "conversion_factor": 1,
+        }
+
+        response = self.client.post(url, data=json.dumps(payload), content_type='application/json')
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.json()['reading'], 10)
+
+        # now delete the item and verify that there are no more readings in the database
+        detail_url = reverse('api:v3:property-meter-readings-detail', kwargs={'property_pk': property_view.id, 'meter_pk': meter_pk, 'pk': '2022-01-05T05:00:00Z'})
+        response = self.client.get(detail_url, content_type='application/json')
+        self.assertEqual(response.status_code, 200)
```

Comment:
what are these lines doing for the test?

Reply:
Verifies that the data are returned, but with just a 200. Nothing fancy.